CMakeLists.txt

```diff
@@ -156,15 +156,52 @@ if (EXISTS .gitignore)
 
   set(CPACK_SOURCE_GENERATOR ZIP)
   set(CPACK_SOURCE_IGNORE_FILES ${ignored_files})
-  set(CPACK_SOURCE_PACKAGE_FILE_NAME cppformat-${CPPFORMAT_VERSION})
```

Comment:
I replaced it with `CPACK_PACKAGE_NAME` which is used for all generators, not only the source ZIP generator. Binary packages generated via `make package` are now also named correctly.

However, it seems like I have missed that it now generates  `cppformat-<version>-Source.zip` instead of ` cppformat-<version>.zip` which definitely was not intended. Good catch!